Examples/test-suite/fortran/Makefile.in

```diff
@@ -0,0 +1,68 @@
+#######################################################################
+# Makefile for Fortran test-suite
+#######################################################################
+
+LANGUAGE	= fortran
+FC      	= @FC@
+SCRIPTSUFFIX= _runme.f90
+
+FORTRANOBJEXT = o
+
+FORTRAN_SO = @FORTRAN_SO@
+FCFLAGS = @FCFLAGS@
+CCSHARED=	@CCSHARED@
+
+srcdir         = @srcdir@
+top_srcdir     = @top_srcdir@
+top_builddir   = @top_builddir@
+abs_top_srcdir = @abs_top_srcdir@
+
+CPP_TEST_CASES = \
+	fortran_bindc
+
+CPP11_TEST_CASES = \
+	fortran_shared_ptr
+
+C_TEST_CASES = \
+	fortran_array_typemap \
+	fortran_bindc_c \
+	fortran_bindc_all \
+	fortran_enums \
+
+include $(srcdir)/failing-cpp-tests.mk
+include $(srcdir)/failing-c-tests.mk
```

Comment:
Please remove the included files and keep these in one file

Reply:
Will do; during development I was using a script that would track the failing tests and write them to these files (so that I could detect regressions with `git diff`).